openedx/features/course_experience/tests/views/test_course_outline.py

```diff
@@ -37,20 +37,23 @@ def setUpClass(cls):
             with cls.store.bulk_operations(course.id):
                 chapter = ItemFactory.create(category='chapter', parent_location=course.location)
                 section = ItemFactory.create(category='sequential', parent_location=chapter.location)
-                ItemFactory.create(category='vertical', parent_location=section.location)
-                course.last_accessed = section.url_name
-
+                vertical = ItemFactory.create(category='vertical', parent_location=section.location)
+            course.children = [chapter]
```

Comment:
@andy-armstrong: I saw different ways to set up the children, but none of them seem great (including this).  Feel free to suggest the right way of handling this.